lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -43,6 +43,7 @@ using namespace circt;
 using namespace firrtl;
 using circt::comb::ICmpPredicate;
 using hw::PortDirection;
+using llvm::Optional;
```

Comment:
Is this necessary? IIRC LLVM is deprecating llvm::Optional so it would be nice to use std::optional instead.

Reply:
Yeah great point! Removed that and replaced the one use with `std::optional`.